docs/user/unattended/profile/users.md

```diff
@@ -67,8 +67,10 @@ Make sure the selected hashing method is supported by the target system, differe
 support different set of methods.
 
 :::warning
+
 Do not use any DES or MD5 based algorithms, these are considered insecure. Check `man 5 crypt`
 manual page for details about the hashing methods and their strength.
+
```

Comment:
NP: `openssl.rpm` is IMHO more likely to be installed than `whois.rpm`

Reply:
I did not write the original text, so I just moved from elsewhere. For me it is fine is someone comes with a proposal. :-)